LiteFarmOrg/LiteFarm
# Addition of empty file error message on the FileUpload component

To test,


1. open http://localhost:3000/map
2. select an empty CSV file and check the error message.

For more details: 
https://lite-farm.atlassian.net/browse/LF-2616
